.circleci/config.yml

```diff
@@ -7,62 +7,76 @@ version: 2.1
 
 commands:
   go_save_cache:
-    description: "Save $GOHOME directory into the cache for the actual revision"
+    description: Save $GOPATH directory as workspace
     steps:
-      - save_cache:
-          key: env-{{ .Revision }}
+      - persist_to_workspace:
+          # Must be an absolute path, or relative path from working_directory. This is a directory on the container which is
+          # taken to be the root directory of the workspace.
+          root: /home/circleci/go
+          # Must be relative path from root
           paths:
-            - /go
-            - /usr/local/bin/golangci-lint
-            - /usr/local/kubebuilder
-            - /usr/local/bin/helm
+            - .
   go_restore_cache:
-    description: "Restore golang cache"
+    description: Restore golang cache
     steps:
-      - restore_cache:
-          key: env-{{ .Revision }}
+      - attach_workspace:
+          # Must be absolute path or relative path from working_directory
+          at: /home/circleci/go
   python_install_requirements:
-    description: "Install requirements in requirements.txt using pip"
+    description: Install requirements in requirements.txt using pip
     steps:
       - run:
           name: Install Python requirements
           command: pip3 install -r tasks/requirements.txt
   alpine_install_git:
-    description: "Install required dependencies"
+    description: Install required dependencies
     steps:
       - run:
           name: Install git
-          command: apk update && apk add git openssh
+          command: apk add --no-cache git openssh
+  ebpf_builder_save_cache:
+    description: Save ebpf-builder tar for later use
+    parameters:
+      path:
+        type: string
+        default: bin/ebpf-builder/ebpf-builder.tar.gz
+        description: ebpf-builder tar location
+    steps:
+      - run: docker save ebpf-builder-amd64 -o <<parameters.path>>
+      - save_cache:
```

Comment:
Here we want to store this for long term and reuse it in next builds, it aims to speed up the docker build time, like it does already locally.